homeassistant/components/google_assistant/helpers.py

```diff
@@ -16,10 +17,19 @@ def __init__(self, code, msg):
 class Config:
     """Hold the configuration for Google Assistant."""
 
-    def __init__(self, should_expose, allow_unlock, agent_user_id,
+    def __init__(self, should_expose, allow_unlock,
                  entity_config=None):
         """Initialize the configuration."""
         self.should_expose = should_expose
-        self.agent_user_id = agent_user_id
         self.entity_config = entity_config or {}
         self.allow_unlock = allow_unlock
+
+
+class RequestData:
+    """Hold data associated with a particular request."""
+
+    def __init__(self, config, user_id, request_id):
```

Comment:
Now I understand the confusion. I didn't realize that the direct connection of Google Assistant can leverage different users in HA. That's pretty cool.